.jenkins/pipelines/standalone/azure-sdk-tests.Jenkinsfile

```diff
@@ -79,6 +79,13 @@ pipeline {
                    """
             }
         }
+        stage('Setup AZ CLI') {
+            steps {
+                sh """
+                    ${JENKINS_SCRIPTS}/azure-sdk/install-azure-cli.sh
```

Comment:
I assume that because we're using the Jenkins plugins to authenticate/interact with Azure, Azure CLI is only needed for the SDK tests?

Reply:
It looks like the VM will not be authenticated as part of the Azure SDK tests. I've added that in as part of setup now. Thanks!